### PR TITLE
Doc: TLS config

### PR DESCRIPTION
**Pull request by urso**
enhance and document TLS configuration

### CODE REVIEW

**Comment by urso** [comment]
Update documentation for #100

**Comment by ruflin** [comment]
@urso Looks like this config change is also a BC change. Should be added to CHANGELOG.md. LGMT,

**Comment by urso** [comment]
good point

File: etc/libbeat.yml [hunk 1]
```diff
@@ -65,11 +56,32 @@ output:
...
-    # Controls whether the client verifies server certificates and host name.
-    # If tlsinsecure is set to true, all server host names and certificates will be
-    # accepted. In this mode TLS based connections are susceptible to
-    # man-in-the-middle attacks. Use only for testing.
-    #tlsinsecure
+    # tls configuration
+    #tls:
+      # If disabled is set to true, the tls section will be ignored and the
+      # host its certificate authorities will be used.
+      #disabled: true
```
Comment (ruflin) [inline etc/libbeat.yml]: I prefer to have "positiv" configs like "enabled", but that is a discussion we should have in the code base, not the docs.

Comment (ruflin) [inline etc/libbeat.yml]: I prefer to have "positiv" configs like "enabled", but that is a discussion we should have in the code base, not the docs.

Comment (urso) [inline etc/libbeat.yml]: it's kind of a 'hidden' option to disable tls (similar effect to commenting out the section). Due to false being the default value for booleans I opted for not naming it enabled. We can remove this flag as well.

**Pull request merged**